src/common/error/errorHelper.ts

```diff
@@ -25,7 +25,7 @@ export class ErrorHelper {
     }
 
     public static getNestedWarning(innerError: Error, message: string, ...optionalArgs: any[]): NestedError {
-        return new NestedError(-1, message, innerError, InternalErrorLevel.Warning);
+        return new NestedError(-1, message, innerError, null /* extras */, InternalErrorLevel.Warning);
```

Comment:
I think the best approach would be instead of having multiple optional positional arguments, go with a named-option-bag approach, so you'd pass in `{level: InternalErrorLevel.Warning}` etc, but that's for another PR